src/app/auth/callbackPage/CallbackPage.tsx

```diff
@@ -28,12 +28,13 @@ const CallbackPage = ({ history }: CallbackPageProps) => {
           },
         });
 
-        const { name, email, sub } = user.profile;
+        const { name, email, sub, rest } = user.profile;
 
         client.writeData({
           data: {
             currentUser: {
               __typename: 'CurrentUser',
+              ...rest,
```

Comment:
I would have preferred that we model this properly here:

https://github.com/City-of-Helsinki/berth-reservations-ui/blob/a849141a4973f8f6cd954e85291b068e61b62d0b/src/app/apolloClient.ts#L13-L17

So that the content of the user is not oblique. 

Seems like we are now using two different data models for this data type:

https://github.com/City-of-Helsinki/berth-reservations-ui/blob/a849141a4973f8f6cd954e85291b068e61b62d0b/src/app/apolloClient.ts#L80

Reply:
Fixed